linux.md

```diff
@@ -28,11 +28,23 @@ Make sure that you're running the 2nd-newest OS version or the newest version -
    corepack enable
    corepack prepare pnpm@latest --activate
    pnpm setup
+   source ~/`[[ $SHELL == *"zsh" ]] && echo '.zshrc' || echo '.bashrc'`
```

Comment:
@ProchaLu in [comment #2270597504](https://github.com/upleveled/system-setup/pull/91#issuecomment-2270597504)

> ## Linux
> I tested the System Setup Guide with @Eprince-hub [ubuntu-linux-github-actions](https://github.com/Eprince-hub/ubuntu-linux-github-actions) project. I had to install `pnpm` using this script
> 
> ```shell
> wget -qO- https://get.pnpm.io/install.sh | ENV="$HOME/.bashrc" SHELL="$(which bash)" bash -
> ```

this is not the same installation method we are using above

Reply:
I created a new Dockerfile that uses only a bash shell and installs the following software:

```
apt-get install -y build-essential git nodejs
```

I was able to recreate the installation steps for Upleveled Preflight. During testing, I found out that before installing Preflight, you need to source the shell configuration file to set up PNPM.


```bash
➜  ubuntu-linux-github-actions git:(main) ✗ docker run -it corepack-pnpm-test
root@3a10a6a19fb8:/# pnpm -v
bash: pnpm: command not found
root@3a10a6a19fb8:/# corepack enable
root@3a10a6a19fb8:/# corepack prepare pnpm@latest --activate
Preparing pnpm@latest for immediate activation...
root@3a10a6a19fb8:/# pnpm setup
Appended new lines to /root/.bashrc

Next configuration changes were made:
export PNPM_HOME="/root/.local/share/pnpm"
case ":$PATH:" in
  *":$PNPM_HOME:"*) ;;
  *) export PATH="$PNPM_HOME:$PATH" ;;
esac

To start using pnpm, run:
source /root/.bashrc
root@3a10a6a19fb8:/# pnpm add --global @upleveled/preflight
 ERR_PNPM_NO_GLOBAL_BIN_DIR  Unable to find the global bin directory

Run "pnpm setup" to create it automatically, or set the global-bin-dir setting, or the PNPM_HOME env variable. The global bin directory should be in the PATH.
root@3a10a6a19fb8:/# source ~/`[[ $SHELL == *"zsh" ]] && echo '.zshrc' || echo '.bashrc'`
root@3a10a6a19fb8:/# pnpm add --global @upleveled/preflight
Packages: +208
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 208, reused 0, downloaded 208, added 208, done

/root/.local/share/pnpm/global/5:
+ @upleveled/preflight 7.0.9

Done in 2.2s
root@3a10a6a19fb8:/# 
```